kits/daisy/src/robot/hexapod.hpp

```diff
@@ -91,6 +91,8 @@ class Hexapod
   void clearLegColors();
   void setLegColor(int leg_index, uint8_t r, uint8_t g, uint8_t b);
 
+  Eigen::Vector3d getGravityDirection() const { return gravity_direction_; }
```

Comment:
return const reference

Reply:
I'd prefer to return a copy of the variable, so any thread-safety issues are isolated to this class.  Is that reasonable?  If it was a heavyweight class, I'd be more concerned.

Reply:
If any issues arise, it's because we haven't added enough documentation that it is not thread safe